spec/controllers/api/v4/identifiers_controller_spec.rb

```diff
@@ -1,16 +1,15 @@
 require 'rails_helper'
 
-RSpec.describe Api::V4::AppIdentifierVisitorConfigsController do
```

Comment:
these tests are basically a mash-up of the v1 identifier controller spec and the v4 app visitor config spec